flexmeasures/data/models/generic_assets.py

```diff
@@ -58,15 +58,15 @@ class GenericAsset(db.Model, AuthModelMixin):
 
     def __acl__(self):
         """
+        All logged-in users can read.
```

Comment:
Only true for **public** generic assets, right?

Reply:
You're correct, I stopped short at that sentence.